util/package.js

```diff
@@ -1,5 +1,20 @@
 'use strict';
 
+var allStars = require( 'all-stars' );
+var objectAssign = require( 'object-assign' );
+
+/**
+ * Check all-stars db for given person and assign to person if found.
+ *
+ * @param  {Object} person object representing author or maintainer
+ *
+ * @return {Object}        same object given, possibly modified or augmented
+ */
+function getAllStar( person ) {
+  // override properties from all-stars if available
+  return objectAssign( person, allStars( person ) );
+}
```

Comment:
I'm really sorry about the confusion. :(

> My first approach explicitly added string properties npm, github, and twitter to the person object. But it was using a few if statements. You suggested using extend or assign instead.

Using a lot of `if/else` statements in a row doesn't look good in my opinion and is not really extendable. Hopefully you agree there?

> How many of the properties listed above would you like? Are you suggesting that the summary() function in all-stars return an object (instead of a string) with the following properties?

Aaah I see - here is the confusing coming from. I thought it is returning an object containing strings.
E.g.

```
{
  name : 'John Doe',
  email : 'john@doe.io',
  npm : 'johndoe',
  github : 'johndow',
  twitter : '@johndoe'
```

```
var allStar = allStars( person );
return allStar ? objectAssign( person, allStar.summary() ) : person;
```

This war pretty much what I had in mind, yes. :bowtie: 

********************************
For all-stars I see 2 requests here ( if you agree ):
- provide a summary object
- improve documentation to make return clearer.

What do you think?